numpy/distutils/exec_command.py

```diff
@@ -249,14 +249,25 @@ def _exec_command(command, use_shell=None, use_tee = None, **env):
         proc = subprocess.Popen(command, shell=use_shell, env=env,
                                 stdout=subprocess.PIPE,
                                 stderr=subprocess.STDOUT,
-                                universal_newlines=True)
+                                universal_newlines=False)
```

Comment:
Can you add a comment here explaining what effect this has on whether the return value is `unicode` / `bytes`? I remember this was discussed in detail in the last PR, and it would be good to encapsulate the conclusions in the source code.